api/lib/gateways/UHT-Contacts/SearchApi.js

```diff
@@ -26,7 +27,8 @@ module.exports = options => {
     return await rp(`${baseUrl}/api/v1/households`, {
       method: 'GET',
       headers: {
-        'X-API-Key': apiKey
+        'X-API-Key': apiKey,
+        'Authorization': `Bearer ${token}`
```

Comment:
Nice!
I know this is temporary but was wondering if there were scenarios where having both the API key and authorisation was needed in a request.

Reply:
I can't think of a reason why you would have two forms of authentication. That's not to say there isn't one though..